ehitchhiking-ui/src/app/shared/components/rate-passangers-modal/rate-passangers-modal.component.ts

```diff
@@ -0,0 +1,37 @@
+import {Component} from '@angular/core';
+import {MatDialogRef} from '@angular/material';
+
+export interface Passanger {
+  id: number;
+  name: string;
+  rating: number;
+}
+@Component({
+  selector: 'app-rate-passang-screen',
```

Comment:
selector name (without app-) should be equal to component name

Reply:
this selector is redundant at all
fixed